Makefile

```diff
@@ -77,6 +77,10 @@ start-cockroach:
 	docker exec -d prisma-engines_cockroach_1 cockroach sql --insecure -e "set cluster setting sql.defaults.default_int_size = 4;"
 	docker exec -d prisma-engines_cockroach_1 cockroach sql --insecure -e "set cluster setting sql.defaults.serial_normalization = 'sql_sequence_cached';"
 
+dev-cockroach: start-cockroach
+	echo 'cockroach' > current_connector
```

Comment:
is this right? i copied `pgbouncer`, but i think the connector should actually be `cockroach`?

Reply:
it also seems to skip a lot of tests (e.g. writes::nested_mutations::already_converted::nested_connect_inside_create::connect_inside_create::p1_req_to_cm_connect_by_uniq_140) -- is this deliberate?

Reply:
fixed!